src/utils.js

```diff
@@ -108,6 +108,18 @@ const validateDeviceId = function validateDeviceId(deviceId) {
   return true;
 };
 
+const validateTransport = function validateTransport(transport) {
```

Comment:
is this checked on initialization? In case it comes in bad

Reply:
it's not but it should be, good call